### PR TITLE
Clarification of borrowck

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
Same as other recent PR with abbreviation

### CODE REVIEW

File: src/borrow-splitting.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ let c2 = &x.c;
 println!("{} {} {} {}", a, b, c, c2);
 ```
 
-However borrowck doesn't understand arrays or slices in any way, so this doesn't
-work:
+However the borrow checker doesn't understand arrays or slices in any way, so
```
Comment (JohnTitor) [inline src/borrow-splitting.md:26]: Why did you change the term even though we explained `dropck` in the previous section? Note that `dropck` is a well-used term on Rust and it's unnecessary to say "borrow checker" so aggressive.

Comment (Arthur-Milchior) [reply src/borrow-splitting.md:26]: Hi,

I assume you meant "borrowck" since this PR don't touch "dropck".

I used the same heuristic than in other PR I did recently that it's nice to avoid abbreviation, unless the texts become significatively longer. I love this rule in general, however I admit it is useless if other documents don't follow it, and in this case introducing it once and then using it to let the reader be used to it makes a ton of sense. 
If we go with the second notion, then I'm fine changing the PR so that it simply adds "(a.k.a. borrowck )", so at least it'll be introduced once.

Comment (JohnTitor) [reply src/borrow-splitting.md:26]: > I assume you meant "borrowck" since this PR don't touch "dropck".

Yeah, you're right.


> I used the same heuristic than in other PR I did recently that it's nice to avoid abbreviation, unless the texts become significatively longer. I love this rule in general, however I admit it is useless if other documents don't follow it, and in this case introducing it once and then using it to let the reader be used to it makes a ton of sense. If we go with the second notion, then I'm fine changing the PR so that it simply adds "(a.k.a. borrowck )", so at least it'll be introduced once.

I think not using "borrowck" only on this doc is somewhat weird, so yes, let's just adding to the first one.

**Comment by Arthur-Milchior** [comment]
Thenk I've just added "a.k.a. borrowck" to be explicit on first use, and kept the standard abbreviation every where else.

By the way, thanks a lot for accepting most of my changes and taking time to discuss the ones that are not valid. And obviously thanks for taking care of this very interesting documentation

**Review by JohnTitor — APPROVED** [review]
Thanks for making those changes too :)
